### DISCUSSION TITLE
support for original xbox controller?

### DISCUSSION THREAD

**Discussion by soniciso1** [cathery/sys-con#233] [Controller requests]
any way you can add support for the og duke and S controllers? once you get an adapter or splice it with a usba it works on pc...

**Comment by doctor-amaton** [comment]
I'm supossed to have one of those around there somewhere. If I can find it I'll look into it, but don't hold your breath.

For visibility there's already an userspace driver set up for this with Linux support: [xboxdrv](https://xboxdrv.gitlab.io/).

EDIT: I was looking into this and I found that the functionality was already there, but is not mentioned in the README.md or basically anywhere.

The functionality was added on fc0d3136e373d2fc693277ecbf466a8166e393c7
I need to test it out and see if it works though, and for that I need to get my USB Adapter... or... build one...

**Reply by megagamer870-github** [reply]
does it work? im gonna buy a shit ton of adaptors if it does

**Reply by doctor-amaton** [reply]
Haven't tested it XD I also haven't got an adapter for that yet. I made one forever ago but no idea where that is...

**Reply by megagamer870-github** [reply]
> s

i got myself one and it just crashed atmosphere... im gonna try reinstalling sys con and see if it works but it is really weird how this is the only place on the internet where anybody is acknowledging the existence of og xbox controller support on sys con. or its just buried by tutorials of how to get the xbox one controller working on sys con

**Reply by megagamer870-github** [reply]
yeah it still didnt work. either its gonna be fixed later or its something on my end that needs to be fixed
